Kudu.Services/Kudu.Services.csproj

```diff
@@ -11,11 +11,6 @@
     <DocumentationFile>bin\Debug\Kudu.Services.XML</DocumentationFile>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <TargetFrameworkProfile />
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <NoWarn>1591</NoWarn>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
```

Comment:
That looks like a random diff that probably should not be included.

Reply:
Uh, how do I pull the two files out of the change list?

Reply:
Several ways, but one is to just push another commit that restores them. It all gets squashed in the end anyway.